data/duke.txt

```diff
@@ -0,0 +1,2 @@
+T,N,borrow
+E,N,aaa,aaa
```

Comment:
Ideally, this file should be ignored using the .gitignore file, so that git does not track such trivial changes

Reply:
Oh thanks